tests/storage/hotplug.go

```diff
@@ -1156,17 +1166,23 @@ var _ = SIGDescribe("Hotplug", func() {
 	Context("virtctl", func() {
 		var (
 			vm *v1.VirtualMachine
+			sc string
 		)
 
 		BeforeEach(func() {
+			var exists bool
```

Comment:
Maybe check and skip if RWOFilesystem is blank. But that should basically never happen.